components/landing/topCollections/TopCollections.vue

```diff
@@ -10,7 +10,7 @@
         <NeoButton
           v-for="{ value, label } in timeRanges"
           :key="value"
-          class="control flex-1 p-0 min-w-[70px] max-md:w-full md:min-w-[98px] px-4 mobile-padding"
+          class="!min-w-[70px] md:!min-w-[110px] control flex-1 p-0 max-md:w-full px-4 mobile-padding"
```

Comment:
```suggestion
          class="control flex-1 p-0 !min-w-[70px] max-md:w-full md:!min-w-[98px] px-4 mobile-padding"
```

why did you change `md:!min-w-[110px]` width amount ?